emission/tests/modellingTests/modellingTestAssets.py

```diff
@@ -1,35 +1,145 @@
 import random
 from typing import Optional, Tuple, List, Dict
 from uuid import UUID
-import emission.analysis.modelling.trip_model.trip_model as eamtm
+import emission.analysis.modelling.trip_model.greedy_similarity_binning as eamtg
+import emission.tests.modellingTests.modellingTestAssets as etmm
 import emission.core.wrapper.confirmedtrip as ecwc
 
 import emission.core.wrapper.entry as ecwe
 import time 
 import math
 
+def setModelConfig(metric,threshold,cutoff,clustering_way,incrementalevaluation):
+    """
+    TODO: Write about each parameter to the function
+    pass in a test configuration to the binning algorithm.
+    
```

Comment:
I am not sure that it is worth having a wrapper (`setModelConfig`) for just two lines of code. It doesn't seem to indicate an actual improvement over just having `model_config = {...}` with essentially the same parameters multiple times.

My comment around pulling out the code https://github.com/e-mission/e-mission-server/pull/933#discussion_r1312355017 was to avoid having the same model configs created over and over again. Each of your tests created (and creates) `model1`, `model2` and `model3` with the same configurations. you should put them into `setup` or create a wrapper function that creates all the configs.

Reply:
Removed in  commit f5944ccef1140db347211c93b851b8864c0362ee